app/index.html

```diff
@@ -0,0 +1,41 @@
+<html>
+	<head>
+	</head>
+	<body>
+		<h2>
+			Welcome to the band name generator
```

Comment:
Looks like you're not using two-space indentation - that makes it easier to read on GitHub. There should be a little menu in the bottom-right of your Sublime window; choose 2 spaces (instead of 4 spaces or tab). You can also add a user setting that will default to 2 spaces.